models/v1beta1/invitation/invitation.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	"github.com/gofrs/uuid"
+	"github.com/lib/pq"
```

Comment:
@aabidsofi19, unless we Hari  eat a decision to switch for ORMs, we’ll need to remove use of pm and cease its future use.